interfaces/builtin/modem_manager.go

```diff
@@ -117,6 +117,19 @@ dbus (receive, send)
     path=/org/freedesktop/ModemManager1{,/**}
     interface=org.freedesktop.DBus.*
     peer=(label=unconfined),
+
+# Allow accessing logind services to properly shutdown devices on suspend
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/login1
+    interface=org.freedesktop.login1.Manager
+    peer=(label=unconfined),
```

Comment:
The network-manager interface has similar rules, but limited receives to PrepareForSleep, SessionNew and SessionRemoved. What is login1.Manager sending to ModemManager?

Reply:
It looks like MM only listens for `PrepareForSleep`, and uses `Inhibit` to create a delay lock to give it time to tear down its connections.  It doesn't look like it's listening for any other signals from logind.